apps/full-site-editing/full-site-editing-plugin/readme.txt

```diff
@@ -0,0 +1,41 @@
+=== Full Site Editing ===
+Contributors: automattic, get_dave, marekhrabe, mppfeiffer, obenland
+Tags: block, blocks, editor, gutenberg, page
+Requires at least: 5.0
+Tested up to: 5.2
+Stable tag: 1.0
+Requires PHP: 5.6.20
+License: GPLv2 or later
+License URI: https://www.gnu.org/licenses/gpl-2.0.html
+
+Enhances your page creation workflow within the Block Editor.
+
+== Description ==
+
+This plugin comes with a custom block to display a list of your most recent blog posts, as well as a template selector
+to give you a head start on creating new pages for your site.
+
+
+== Installation ==
+
+1. Upload the plugin files to the `/wp-content/plugins/full-site-editing` directory, or install the plugin through the WordPress plugins screen directly.
+1. Activate the plugin through the 'Plugins' screen in WordPress.
```

Comment:
These are all 1. bullet points.

Reply:
I adopted that from the [readme example](https://wordpress.org/plugins/readme.txt). Markup converts those into an ordered list